client/offchain/src/api/http.rs

```diff
@@ -699,7 +705,8 @@ mod tests {
 			// not be enough).
 			fdlimit::raise_fd_limit();
 
-			let (api, worker) = http();
+			let hyper_client = Arc::new(HyperClient::builder().build(HttpsConnector::new()));
```

Comment:
Actually just realised that it might be good to put it into `lazy_static` and that would most likely fix the `fdlimit` issue described above.

Reply:
https://github.com/paritytech/substrate/pull/6454/commits/7af97498a2383b5d7405e27823db8fd97245da41 :D

Reply:
Revert previous commit (https://github.com/paritytech/substrate/pull/6454/commits/fc822c4ae132e99b42314dd3ac1fcaebdea83e81) and added a new commit to use lazy_static only for tests https://github.com/paritytech/substrate/pull/6454/commits/531353412e581114405fdd9355d344ce930249c6 :)